### PR TITLE
Add draw_polygon tests

### PR DESCRIPTION
**Pull request by davidtheclark**

### CODE REVIEW

**Comment by mcwhittemore** [comment]
@davidtheclark is this ready for review?

**Comment by davidtheclark** [comment]
@mcwhittemore No, I abandoned it part way to work on the events. Will pick it up again after the events are done.

**Comment by davidtheclark** [comment]
@mcwhittemore now this is ready for review.

File: test/draw_polygon.test.js [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+    },
+    geometry: {
+      type: 'LineString',
+      coordinates: [[0, 0], [0, 10]]
+    }
+  }, 'a line string is sent to the callback');
+  t.end();
+});
+
+test('draw_polygon render, inactive, with only two vertices', t => {
```
Comment (mcwhittemore) [inline test/draw_polygon.test.js]: This is inactive because there is no `id` not because the feature only has two vertices.

Comment (davidtheclark) [inline test/draw_polygon.test.js]: Do you think we should remove this test? Or leave as-is?

Comment (mcwhittemore) [inline test/draw_polygon.test.js]: it looks like its tested again lower down, so lets drop this.

**Pull request merged**